src/server/Document/__snapshots__/component.test.jsx.snap

```diff
@@ -78,6 +78,22 @@ exports[`Document Component should render APP version correctly 1`] = `
   lang="test"
 >
   <head>
+    <script>
+      
+            window.__reverb = {};
+            window.__reverb.__reverbLoadedPromise = new Promise((resolve, reject) =&gt; {
+              window.__reverb.__resolveReverbLoaded = resolve;
+              window.__reverb.__rejectReverbLoaded = reject;
+            });
+
+            window.__reverb.__reverbTimeout = setTimeout(() =&gt; {
+              window.__reverb.__rejectReverbLoaded();
+            }, 5000);
+    </script>
+    <script
+      async=""
+      src="https://mybbc-analytics.files.bbci.co.uk/reverb-client-js/reverb-3.9.2.js"
```

Comment:
Same as above

Reply:
This is a snapshot file and as such it will retain the URL.